src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard
+          movie={movie}
+        />
```

Comment:
Wrap if you have > 2 params || length > 80
```suggestion
        <MovieCard movie={movie} key={movie.imdbId} />
```